### ISSUE TITLE
Panels card

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/genepanel.iobio#198]
Ok, I think it might be difficult to really explain the chart, and make it look nice etc. What do you think about just making this look like all the other cards?

![asset 19](https://user-images.githubusercontent.com/523111/43010864-b02a750c-8c0f-11e8-9f9b-c1bf7ee7fa4f.png)

So, the user can see a list of all the gene panels, and attached to each panel, we have the number of genes on the panel, and the number of conditions associated with the panel. I'll let you play around, but maybe instead of the bars, we just provide the numbers (depending on available room). The hope is that a user will open this and appreciate that the panel is very general (e.g. is associated with many disorders), or very specific (only for a very small number of disorders). If you can think of a way of conveying this message (maybe we use colours somehow to identify specific, moderate, or general or something - and we define how the ranges etc.), that would be great! 

What do you think? We might need to experiment with whether you can use bars, stacked bars.

![asset 20](https://user-images.githubusercontent.com/523111/43011180-b3689fe0-8c10-11e8-8084-0988a4f9d477.png)

**Comment by AlistairNWard** [comment]
Here would be another way of going. This is really quick; I'm just throwing around ideas!

Next to each panel we give the user the number of genes and the number of conditions associated with the panel. We define "specific", "moderate", and "general" (or whatever terms we decide) to be e.g. <10, 10<x<30, >35 and give them a colour. (We could add a settings wheel at the top right where the user can modify these definitions?) The user could then deselect any panel they don't want, or just select/deselect all general panels etc. The explanation would concisely convey this, and point the user to the settings to update the definitions.

![Asset 20.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5750332babe50b1300c53bf6/125516c66e82c728ace21e0d46c987892696c3fbdfabf301b44fff9e2c4533317346f62a2980e771e6402e07095f1ca941585e16fefad823a4e22e6c981e5ab38532022d8c6853beb2901eb80bbb25884d84b6bf8c44c1ef3b25f5c6.png)

**Comment by AlistairNWard** [comment]
Yes, I think it best to keep them separate. I think it is a clearer connection that the disorder/inheritance anyway, so I don't think that's a problem.

Maybe in a v2 release, we can add tooltips to these cards so you can be looking at a panel and see who the vendor is. Also, if we deselected some panels, and that removed all panels from a particular vendor, we could have a modal pop-up that tells the user. Not anything intrusive that they have to click away, just a notification. We don't need that for a v1 release though,

**Comment by AlistairNWard** [comment]
Tony's preference was to include the bars. Do you think there is enough room to include them?

Also, could we experiment with including the idea of specific, moderate, and general panels and filters?

**Comment by adityaekawade** [comment]
How do you want the filtering for specific - moderate - general panels? You had mentioned  <10, 10<x<30, >35 based on the number of conditions. But the number of conditions is a small number.

**Comment by adityaekawade** [comment]
I will try to include the bars, but I think they would be small. Also, I can try by reducing the width and the font size in the panel's column.

**Comment by AlistairNWard** [comment]
If we add a settings cog in the menu bar, we could make the settings definable by the user. Also, we could define the panels as the number of genes on them rather than the number of conditions. Then we could use the moderate as 10<x<50? Specific and general would be less than or greater than these.
